Update FAQs (#582)
mattermost/mattermost-handbook: operations/finance/spending-company-money/how-to-use-accrualify/frequently-asked-questions.md

## FAQs

**How do I submit an off-cycle budget?**

> * To submit an off-cycle budget, please refer to the Handbook under the [Budget section](https://handbook.mattermost.com/operations/finance/spending-company-money/budget).

Reviewer: ```suggestion
* To submit an off-cycle budget, refer to the Handbook under the [Budget section](https://handbook.mattermost.com/operations/finance/spending-company-money/budget).
```

**Is an off-cycle budget request required before submitting an PO on Accrualify?**

> * Yes, before submitting a PO request on Accrualify, please make sure that your budget spend has been approved by the Finance Team. If a PO request is submitted before a budget request, please expect delays in your request. 

Reviewer: ```suggestion
* Yes, before submitting a PO request on Accrualify, make sure that your budget spend has been approved by the Finance Team. You can reach out by sending an email to spend@mattermost.com. If a PO request is submitted before a budget request is approved, expect delays in your request. 
```

> * Yes, before submitting a PO request on Accrualify, make sure that your budget spend has been approved by the Finance Team. You can reach out by sending an email to spend@mattermost.com. If a PO request is submitted before a budget request is approved, expect delays in your request. 

Reviewer 2: ```suggestion
* Yes, before submitting a PO request on Accrualify, make sure that your budget spend has been approved by the Finance Team. You can reach out by sending an email to spend@mattermost.com. If a PO request is submitted before a budget request is approved, expect delays in your request.
```

**What is the turnaround time for PO request approvals?**

> * Please allow 1-2 weeks for PO request approvals. Each PO request is reviewed by Finance and Legal. Please remember to attach the appropriate legal documents to your PO request so as to not delay the review and approval process.

Reviewer: ```suggestion
* Allow 1-2 weeks for PO request approvals. Each PO request is reviewed by Finance and Legal. Remember to attach the appropriate legal documents to your PO request so the the review and approval process isn't delayed.
```

**Can we purchase from international vendors?**

> * Yes, of course! Please make sure the appropriate legal documents are in English and Mattermost is billed in USD.

Reviewer: ```suggestion
* Yes, of course! Make sure the appropriate legal documents are in English and the billing entity is Mattermost, Inc. We strongly encourage invoices are in USD but can support other currencies if needed. 
```

> **Where can I find my to-do items on Accrualify?**

Reviewer 2: ```suggestion
**Where can I find my to do items on Accrualify?**
```

> * Once you log into your account, you will be brought to your dashboard. From there, you will be able to see your to-do items.

Reviewer 2: ```suggestion
* Once you log into your account, you'll have access to the dashboard. From there, you'll be able to see your to do items.
```

**Where can I find or track the status of my PO request?**

> * Log in to your account and you will see where the current status of your PO request is. If you have any questions, feel free to contact the Finance team in the messaging tool and we will respond within 24 hours. If you do not hear from us within the 24 hours, please give us a gentle nudge.

Reviewer: ```suggestion
* Log in to your account and you will see where the current status of your PO request. If you have questions, feel free to contact the [Finance Team](https://community.mattermost.com/private-core/channels/purchases) and a team member will respond within 24 hours.
```

**What are our standard payment terms?**

* We pay vendors 30 days from the date of receipt of the invoice.

**My vendor has reached out to me and is having issues with the portal. How can we help them?**

> * If your vendor is experiencing issues with the vendor portal, please have them contact support@accrualify.com and copy AP@mattermost.com. Our Accrualify is happy to help directly with your vendor to solve the issue. We want to make sure that your vendor’s experience with the portal is positive and seamless.

Reviewer: ```suggestion
* If your vendor is experiencing issues with the vendor portal, please have them contact support@accrualify.com and copy AP@mattermost.com. Our designated Accrualify Customer Success Team is happy to help directly with your vendor to triage issues. We want to make sure that your vendor’s experience with the portal is positive and seamless.
```
